test/index.js

```diff
@@ -513,6 +549,7 @@ run((assert, comment) => {
 run((assert, comment) => {
   comment('respond to options: individual')
   return test('/animals/1', { method: 'options' }, response => {
+    assert(validate(response.body), 'response does not adhere to json api')
```

Comment:
For these 204's, it may just be that we shouldn't be using this schema. I'll look over my own JSON API implementation to see how I was handling this.